.github/workflows/publish-layer-collector.yml

```diff
@@ -117,6 +117,9 @@ jobs:
           fi
           echo "release_jobs={"architecture": ${architectures}, "aws_region": ${aws_regions}}" | tr -d '[:space:]' >> $GITHUB_OUTPUT
   release-layer:
+    permissions: # required by the reusable workflow
+      contents: read
+      id-token: write
```

Comment:
This workflow doesn't release to Github, but just publish into AWS Lambda layers. So `id-token` permission shouldn't be required (and it is already not used and required)